catalog-template.yaml

```
@@ -713,6 +713,30 @@ entries:
     replaces: rhacs-operator.v4.7.3
     skipRange: '>= 4.6.0 < 4.7.4'
 
+- schema: olm.channel
+  name: rhacs-4.8
+  package: rhacs-operator
+  entries:
+  - *bundle-4-0-0
+  - *bundle-4-1-0
+  - *bundle-4-1-1
+  - *bundle-4-1-2
+  - *bundle-4-1-3
+  - *bundle-4-2-0
+  - *bundle-4-3-0
+  - *bundle-4-4-0
+  - *bundle-4-5-0
+  - *bundle-4-6-0
+  - *bundle-4-7-0
+  - *bundle-4-7-1
+  - *bundle-4-7-2
+  - *bundle-4-7-3
+  - *bundle-4-7-4
```

@msugakov @tommartensen @vikin91 

From a newcomer's perspective, I would expect this to also list all the existing patch release bundles for 4.6, too.

See suggestion.

(I know 4.5 and earlier versions don't list them, but those are out of support, so I'm ignoring them.)

```suggestion
  - *bundle-4-6-0
  - *bundle-4-6-1
  - *bundle-4-6-2
  - *bundle-4-6-3
  - *bundle-4-6-4
  - *bundle-4-6-5
  - *bundle-4-6-6
  - *bundle-4-6-7
  - *bundle-4-7-0
  - *bundle-4-7-1
  - *bundle-4-7-2
  - *bundle-4-7-3
  - *bundle-4-7-4
```

---

There may be something to this suggestion. This catalog format is more flexible and it opens new possibilities whereas my thinking is still mostly formed by limitations imposed by CPaaS.

Doing so might be possible (when we define `replaces:` attributes differently in different channels) but it creates a mismatch that existing `rhacs-4.7` channel won't have 4.6.1-4.6.7 patches and `rhacs-4.8` will. Maybe that's ok and maybe we want all patches in all new channels (there's something about `stable` to consider, but I'll hold it for now).

The main question is why. What would 4.6.1-4.6.7 patches bring for users who'd select `rhacs-4.8` channel?
When they select 4.8, they likely want 4.8 and not install 4.6.7 from it.